src/app/features/auth/form/form.component.ts

```diff
@@ -0,0 +1,144 @@
+import { CommonModule } from '@angular/common';
+import { ChangeDetectionStrategy, Component, inject } from '@angular/core';
+import {
+  FormsModule,
+  NonNullableFormBuilder,
+  ReactiveFormsModule,
+  Validators,
+} from '@angular/forms';
+import { MatButtonModule } from '@angular/material/button';
+import { MatFormFieldModule } from '@angular/material/form-field';
+import { MatIconModule } from '@angular/material/icon';
+import { MatInputModule } from '@angular/material/input';
+import { Store } from '@ngrx/store';
+
+import { AuthResponse, LoginForm } from '../shared/auth.iterfaces';
+import { AuthActions } from '../store/auth.actions';
+import { whitespaceValidator } from './form.validators';
+
+@Component({
+  selector: 'app-form',
+  standalone: true,
+  imports: [
+    MatFormFieldModule,
+    MatInputModule,
+    MatIconModule,
+    MatButtonModule,
+    CommonModule,
```

Comment:
pytanie do Kamila czy potrzebujemy importować CommonModule czy lepiej samo NgIf

Reply:
@akademia-envelo-3/meetek-frontend importujemy tylko to czego potrzebujemy, commonmodule odpuszczamy na rzecz pojedynczych importów